KesaApplication/ViewController.swift

```diff
@@ -10,16 +10,130 @@ import UIKit
 
 class ViewController: UIViewController {
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+    // 4 labels
+    @IBOutlet weak var firstNameLabel: UILabel!
+    @IBOutlet weak var lastNameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet weak var passwordLabel: UILabel!
+    
+    // 4 text fields
+    @IBOutlet weak var firstName: UITextField!
+    @IBOutlet weak var lastName: UITextField!
+    @IBOutlet weak var email: UITextField!
+    @IBOutlet weak var password: UITextField!
+    
+    
+    // When the login button is pressed, ignore all text fields
+    @IBAction func loginButtonPressed(sender: UIButton) {
```

Comment:
I would change the name of the function from 
loginButtonPressed -> loginButtonOnClickListener